tssc/step_implementer.py

```diff
@@ -161,7 +161,7 @@ def __print_data(cls, title, data):
             colalign=("left",)
         ))
 
-    def run_step(self, **kwargs):
+    def run_step(self, global_step_config_defaults, **kwargs):
         """
         Wrapper for running the implemented step.
 
```

Comment:
@kfrankli please add `parameters` section of doc to the funciton doc here.

Reply:
Made changes per request.